### ISSUE TITLE
Bi-directional junctions can get out of sync with their underlying table

### ISSUE THREAD

**Issue by cdo256** [teableio/teable#492]
I have been trying out Teable on app.teable.io for the past day, and it's been working reasonably well, however it has ended up in a state where it's expecting a particular junction table to exist that doesn't. This leads to errors like the following when trying to edit the primary key of either of the affected tables:

```
internal_server_error: An error occurred in $queryRawUnsafe:
Invalid `prisma.$queryRawUnsafe()` invocation:

Raw query failed. Code: `42P01`. Message: `relation "bsePYQ5LBiGxeAyQu26.junction_fld1knDV9BY5h0mTkUB_fldgFzzG4wTywY3mhAR" does not exist`internal_server_error: An error occurred in $queryRawUnsafe:
Invalid `prisma.$queryRawUnsafe()` invocation:
```

I'm not certain what exactly I have done that could have caused this. I have made rather liberal use of multi-multi junctions I have a 'Themes' table which is linked bidirectionally to 'People', 'Publications', 'Research groups', 'Journals' and 'Conferences'. So this could serve as a minor stress-test in this regard. Admittedly I could have used tags instead, however it should not have broken in this way either.

I suspect what may have caused it is the feature where you can nest the creation of records in different tables:

 1. Create a row A in the 'Publications' table.
 2. Click on the 'People' feild of row A to open the selector dialog.
 2. Create an entry in the  'People' table.
 3. Click add a new record, call it B.
 4. In the dialog set the publication to row A that we are in the middle of creating.
 5. Keep going and edit the Themes of B to add a new theme, C.
 6. In the dialog to create the theme C.
 7. Add both A and B to the Publications and People fields of the theme C.
 8. Finalize the 3 records.
 9. Attempt to edit the primary keys of the Publication table or the Theme table, and you should get an error like the one above.

Let me know if you have trouble reproducing this, and I can try to provide more information.

You are welcome to use my base for testing (as long as you don't change my data, of course).

**Comment by tea-artist** [comment]
Thanks for your patient feedback!
I found the wrong data that caused the problem. I will fix this issue completely as soon as possible

I reproduced the problem because changing the field name caused an accident.
Reproduce path: 
1. Create table A and Table B,
2. Create A many-to-many bidirectional link in table A to table B
3. Change the name of field B.A to B.A2
4. The primary key value of table A cannot be changed
